File: alotta-core/src/main/java/liberty/capstone/core/appuser/AppUser.java
package liberty.capstone.core.appuser;

import lombok.Data;

@Data
public class AppUser {
    private String username;
    private String password;
    private String accountType;
    private String firstName;
    private String lastName;
    private String email;
    private String phoneNumber;
    private String zipcode;
}

File: alotta_client/lib/assets/data/appuser.dart
import 'package:flutter/widgets.dart';

class Appuser {
  final String username;
  final String password;
  final String firstName;
  final String lastName;
  final String email;
  final String accountType;

  const Appuser({
    required this.username,
    required this.password,
    required this.firstName,
    required this.lastName,
    required this.email,
    required this.accountType,
  });
}
